### PR TITLE
Feat/image gen microservice

### PR DESCRIPTION
**Pull request by andreasmwenzel**

### CODE REVIEW

File: core/src/image-generation/image-generation.controller.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {Controller, Get, Param} from "@nestjs/common";
+
+import {ImageGenerationService} from "./image-generation.service";
+
+@Controller("image-gen")
```
Comment (0c370t) [inline core/src/image-generation/image-generation.controller.ts:5]: GraphQL, not Rest please

**Review by 0c370t — CHANGES_REQUESTED** [review]
Overall, very well done. One tweak and it is good to go

File: core/src/image-generation/image-generation.controller.ts (hunk 1 above)
Comment (andreasmwenzel) [reply core/src/image-generation/image-generation.controller.ts:5]: Does this actually need a controller? I meant to ask that. I threw this up for really easy testing, but we'll this be called from the web or just imported by the scrim module and called from within nest?

Comment (0c370t) [reply core/src/image-generation/image-generation.controller.ts:5]: That's a good question, and something that we probably need to explore more in the future. Leave it in for now.

**Review by 0c370t — APPROVED** [review]

**Pull request merged**